<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="select_image_button">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>730</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="identify_button">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>730</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Identify</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="file_path">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>650</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Selected image</string>
   </property>
  </widget>
  <widget class="QLabel" name="output_label">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>680</y>
     <width>800</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Current action</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="image_preview">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>40</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>select_button</zorder>
  <zorder>identify_button</zorder>
  <zorder>file_path</zorder>
  <zorder>image_preview</zorder>
  <zorder>out_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>